docs/trace/extending-the-sdk/README.md

```diff
@@ -60,7 +60,103 @@ Exporter to the `TracerProvider` as shown in the example [here](./Program.cs).
 
 ## Instrumentation Library
 
-TBD
+The goal of the OpenTelemetry project is to make every library and application
```

Comment:
Not sure if that's the official goal of OpenTelemetry, might be helpful to put a link here.

Reply:
If both apps and libs are not calling the OpenTelemetry API - who is going to call it? 😄